### PR TITLE
[WIP] button DLS

### PR DESCRIPTION
**Pull request by muhajirdev**
will rename the component into `Button` once we migrate to use new component and delete the old Button

### CODE REVIEW

**Comment by muhajirdev** [comment]
will rename the component into `Button` once we migrate to use new component and delete the old Button

File: src/styles/themes/_dark.scss [hunk 1]
```diff
@@ -34,7 +34,8 @@
   --color-tertiary-medium: #fff;
   --color-tertiary-deep: #fff;
 
-  --color-text-default: #fff;
+  --color-text-default: #384c55;
+  --color-text-inverse: #fff;
```
Comment (tifa2UP) [inline src/styles/themes/_dark.scss:38]: #000

File: src/components/dls/ButtonNew/ButtonNew.module.scss [hunk 2]
```diff
@@ -0,0 +1,173 @@
+.base {
+  border: none;
+  padding: 0 var(--spacing-small);
+  display: flex;
+  align-items: center;
+  font-weight: 500;
```
Comment (tifa2UP) [inline src/components/dls/ButtonNew/ButtonNew.module.scss:6]: let's not define the font weight here. use a font weight token

File: src/components/dls/ButtonNew/ButtonNew.module.scss [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+  }
+}
+.ghost.error {
+  color: var(--color-alert-medium);
+  &:hover {
+    background-color: var(--color-alert-medium);
+  }
+}
+
+.disabled {
```
Comment (tifa2UP) [inline src/components/dls/ButtonNew/ButtonNew.module.scss:142]: how do you make a button disabled?

File: src/components/dls/ButtonNew/Spinner.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames';
+import styles from './Spinner.module.scss';
+
+export enum SpinnerSize {
+  Small = 'small',
+  Normal = 'normal',
+  Large = 'Large',
+}
+type SpinnerProps = {
+  size: SpinnerSize;
```
Comment (tifa2UP) [inline src/components/dls/ButtonNew/Spinner.tsx:10]: this component should be agnostic of button

**Review by tifa2UP — APPROVED** [review]

**Pull request merged**